sources/include/parser/nodetypes.h

```diff
@@ -202,6 +202,8 @@ namespace citygml {
         NODETYPE( BLDG, Lod4TerrainIntersection )
         NODETYPE( BLDG, ConsistsOfBuildingPart )
 
+        NODETYPE( BLDG, lod0RoofEdge )
```

Comment:
他の定義に合わせて大文字スタートでお願いします。
```suggestion
        NODETYPE( BLDG, Lod0RoofEdge )
```